conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/100/package/CodeSystem-IdentifierTypeCS.json

```diff
@@ -1 +1,94 @@
\ No newline at end of file
+{
+	"resourceType": "CodeSystem",
+	"id": "IdentifierTypeCS",
+	"text": {
+		"status": "empty",
+        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Redacted for size</div>"
```

Comment:
interesting mix of tabs and spaces.  i'm assuming you didn't use our FHIRGenerator to generator this.  I was hoping it would use the same approach we use in the other IGs which produces consistently-formatted output

Reply:
I believe the latest is correct -- I ran the formatter again and integrated.

Reply:
conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/100/package/CodeSystem-IdentifierTypeCS.json looks right now to me.

Reply:
ok.  in the couple I worked on we have a class named "ResourceProcessor" that does the narrative text fixup and we write out the formatted output straight from there.  i think it uses spaces and not tab (like the rest of our project), but I think the tabs are probably fine now that they are at least consistent within this project